.github/auto-merge.yml

```diff
@@ -30,10 +26,6 @@
     dependency_name: Microsoft.AspNetCore.StaticFiles
     update_type: all
 
-- match:
-    dependency_name: Microsoft.Build.Utilities.Core
-    update_type: all
-
```

Comment:
https://github.com/Particular/NServiceBus.Persistence.Sql/blob/master/src/ScriptBuilderTask/ScriptBuilderTask.csproj#L29

We only have one other reference, in a private repo, so having this package in the list buys us little anyway.

Reply:
Are we still obligated to support Visual Studio 2017? If we want to stick to version 15 of that dependency why are we not restricting it with range?

Reply:
> Are we still obligated to support Visual Studio 2017? 

https://docs.particular.net/samples/#technology-choices-visual-studio-and-net

> Visual Studio 2017 Update 3.

https://docs.microsoft.com/en-us/visualstudio/releases/2019/servicing#support-for-older-versions-of-visual-studio

> (through April 2027) 

I believe we still support it for now. Making a decision not to seems outside the scope of this PR.

---

> If we want to stick to version 15 of that dependency why are we not restricting it with range?

Feel free to change it.

---

But again, I feel this discussion is moot. We're taking the risk of automerging updates to this package to save one PR.

Reply:
The question of why we would want to still support Visual Studio 2017 is addressed in https://github.com/Particular/Strategy/blob/master/go-to-market/tech-leads.md#ability-to-upgrade.

How the organization makes a decision to drop support isn't clear. I agree with @adamralph it seems to be outside the scope of this PR.